Bindings/Java/OpenSimJNI/Test/testContext.cpp

```diff
@@ -72,7 +72,9 @@ int main()
     LoadOpenSimLibrary("osimJavaJNI");
 
     Model *model = new Model("wrist.osim");
-    OpenSimContext* context = new OpenSimContext(&model->initSystem(), model);
+    SimTK::State* workingState = &model->initSystem();
+    model->realizePosition(*workingState);
+    OpenSimContext* context = new OpenSimContext(workingState, model);
     const ForceSet& fs = model->getForceSet();
```

Comment:
@adamkewley Thanks for looking into this. The GUI never uses a Model realized below Position stage since it needs to display it, requiring position info. Will have to see how it plays with GUI but probably better to call the non-virtual "realizePosition"  method of OpenSimContext at the end of the OpenSimContext constructor and then client code and the GUI doesn't need to change. If the test cases pass feel free to merge. Thank you